utils/casadm.8

```diff
@@ -310,6 +312,36 @@ Period of time between awakenings of flushing thread [ms] (default: 10 ms).
 .B -b, --flush-max-buffers <NUMBER>
 Number of dirty cache blocks to be flushed in one cleaning cycle (default: 128).
 
+.SH Options that are valid with --set-param (-X) --name (-n) promotion are:
+
+.TP
+.B -i, --cache-id <ID>
+Identifier of cache instance <1-16384>.
+
+.TP
+.B -p, --policy {always|nhit}
+Cleaning policy type to be used with a given cache instance.
+
+Available policies:
+.br
+1. \fBalways\fR. Core lines are attempted to be promoted each time they're accessed.
+.br
+2. \fBnhit\fR. Core lines are attempted to be promoted after n accesses.
+
+.SH Options that are valid with --set-param (-X) --name (-n) promotion-nhit are:
+
+.TP
+.B -i, --cache-id <ID>
+Identifier of cache instance <1-16384>.
+
+.TP
+.B -o, --trigger <PERCENTAGE>
+Percent of cache to be occupied before cache inserts will be filtered by the policy.
+
+.TP
+.B -t, --threshold <NUMBER>
+Number of accesses per core line for it to be inserted.
```

Comment:
This sounds more clear to me:
Number of core line accesses required for it to be inserted to cache.

Reply:
Yup, changed to your wording